MemoryMgr.GTA.h

```diff
@@ -268,7 +268,9 @@ namespace Memory
 				if ( auto pao = std::get_if<PatternAndOffset>(&address10) ) return HandlePattern( *pao );
 				else
 				{
-					const uintptr_t addr = *std::get_if<uintptr_t>(&address10);
+					const uintptr_t addr = (std::get_if<uintptr_t>(&address10) != nullptr)
+											? *std::get_if<uintptr_t>(&address10)
+											: std::make_unsigned_t<intptr_t>(*std::get_if<intptr_t>(&address10));
```

Comment:
That might require `<type_traits>` to be included I think.

Reply:
Done